lib/active_record/connection_adapters/odbc_adapter.rb

```diff
@@ -1838,9 +1830,9 @@ def ensure_sequences_table() end
 
       class ODBCColumn < Column #:nodoc:
 
-        def initialize (name, tableName, default, odbcSqlType, nativeType,
-            null = true, limit = nil, scale = nil, dbExt = nil,
-            booleanColSurrogate = nil, nativeTypes = nil)
+        def initialize(name, default, cast_type, odbcSqlType, nativeType, sql_type = nil, null = true,
+          scale = nil, dbExt = nil, booleanColSurrogate = nil, nativeTypes = nil, limit=nil)
```

Comment:
That's some pretty significant changes to the signature. Where does this declaration come from? Is this standard for ActiveRecord db adapters, or is it just specific to this odbc+vertica implementation?